src/sqlite_back.cc

```diff
@@ -293,6 +293,7 @@ void SqliteBack::Bind(boost::spirit::hold_any v, DbTypes type, SqlStatement::Ptr
   CYCLUS_BINDVAL(MAP_STRING_STRING, std::map<std::string CYCLUS_COMMA std::string>);
   CYCLUS_BINDVAL(MAP_STRING_VECTOR_DOUBLE, std::map<std::string CYCLUS_COMMA std::vector<double> >);
   CYCLUS_BINDVAL(MAP_STRING_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::map<int CYCLUS_COMMA double> >);
+  CYCLUS_BINDVAL(MAP_STRING_PAIR_DOUBLE_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::pair<double CYCLUS_COMMA std::map<int CYCLUS_COMMA double> > >);
```

Comment:
Thanks @rwcarlsen. To be frank, I think they both look bad; in such cases,
I default to the style guide.

On Mon, Mar 30, 2015 at 5:16 PM, Robert Carlsen <notifications@github.com>
wrote:

> In src/sqlite_back.cc
> <https://github.com/cyclus/cyclus/pull/1109#discussion_r27437272>:
>
> > @@ -293,6 +293,7 @@ void SqliteBack::Bind(boost::spirit::hold_any v, DbTypes type, SqlStatement::Ptr
> >    CYCLUS_BINDVAL(MAP_STRING_STRING, std::map<std::string CYCLUS_COMMA std::string>);
> >    CYCLUS_BINDVAL(MAP_STRING_VECTOR_DOUBLE, std::map<std::string CYCLUS_COMMA std::vector<double> >);
> >    CYCLUS_BINDVAL(MAP_STRING_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::map<int CYCLUS_COMMA double> >);
> > +  CYCLUS_BINDVAL(MAP_STRING_PAIR_DOUBLE_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::pair<double CYCLUS_COMMA std::map<int CYCLUS_COMMA double> > >);
>
> I'll just do it - not worth the discussion time :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1109/files#r27437272>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192